go.mod

```diff
@@ -61,7 +61,6 @@ require (
 	k8s.io/code-generator v0.16.8
 	k8s.io/kops v1.15.2
 	k8s.io/kubelet v0.16.8
-	k8s.io/kubernetes v1.16.8
```

Comment:
will this line be removed as well https://github.com/weaveworks/eksctl/blob/master/go.mod#L97 ?

Reply:
good question! when I remove it, it gets pulled into `go.sum` anyway at `v1.15.3`, I believe because of `kops`. Perhaps it's safer to leave it?

Reply:
Just my opinion. 

I just enable verbose mode in go build to actually find out _exactly_ what libraries will be used. I don't think k8s.io/kubernetes package is used anywhere. go.mod sometimes tries to be smart while deriving dependencies transitively.

```shell script
$ go clean --cache
$  CGO_ENABLED=0 go build -v ./cmd/eksctl > package.txt 2>&1 
$ wc -l package.txt 
775 package.txt
$ rg "k8s.io/kubernetes" package.txt   
```